app/models/rating.rb

```diff
@@ -0,0 +1,10 @@
+class Rating < ActiveRecord::Base
+  has_many :topics
+  has_many :posts
+
+  def self.update_rating(rating_string)
+    return Rating.none if rating_string.blank?
+
+  enum severity: [ :PG, :PG13, :R ]
```

Comment:
This is the out-of-place `enum` that Levi mentioned. The assignment shows where to place it.